### PR TITLE
Copy config to package lib directory on build

### PR DESCRIPTION
**Pull request by jdetle**
I think this should fix the commuter config issue.

### CODE REVIEW

**Comment by jdetle** [comment]
I think I would need to compile this config to get it to work, which I'm not sure makes sense.

**Comment by jdetle** [comment]
Taking off the WIP tag to let someone know they should look at it, but this is a gross hack that I'm not even sure works. If someone could help me publish a test commuter and configurator update, I would be appreciative.

**Comment by chrnorm** [comment]
@jdetle I'm a quick look at this to see if I can help, I'm not too familiar with the nteract codebase yet but here are some things that I noticed:

- The project-wide `babel.config.js` [actually has overrides for flow and typescript already (see hyperlink)](https://github.com/nteract/nteract/blob/024e7960aec8262fbb03112463a1987602cc244f/babel.config.js#L21-L31), if the overrides work properly there is probably no need for separate `babel.flow.config.js` and `babel.typescript.config.js` files.
- The babel docs specify that for a monorepo `babel.config.js`, you can pass `rootMode: "upward"` as an option to Webpack which will make babel find the config file in the root directory without the need for `require`ing any config files ([more info here](https://babeljs.io/docs/en/config-files#webpack)).

If we added `rootMode: "upward"` to the webpack config and just used `babel.config.js` instead of the two typescript/flow config files, it might solve the problem of finding the config file.

I can open a PR with these changes to see if it would help, but did you have any idea how to test them? As mentioned in #3654, when starting commuter from the git repo the bug isn't there.

**Comment by jdetle** [comment]
Hi @chrnorm, thanks for your comment. I think you're on to something with respect the first bullet point. If you can open a PR that successfully removes extraneous config that would be warmly welcomed.  The second bullet point seems like it won't actually work for fixing the issue at hand, which is that when the package is built and shipped, the root level config files can't exist by design.

There are a few issues that are getting in the way of having a clean environment to test this. We should be able to build a production version of commuter with the production version of all our local packages that it depends on and it should "just work", but this typescript conversion seems to be causing some issues. I have some work to get to before I can really deep dive on assisting in recovery efforts. 

With respect to the commuter config issues: I think using `files` in the package.json of webpack-configurator with the webpack config paths in the package.json is probably a better solution than all of my work that I put in here.

File: packages/webpack-configurator/copy_config.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir lib
+cp ./../../babel.flow.config.js ./lib/babel.flow.config.js
+cp ./../../babel.typescript.config.js ./lib/babel.typescript.config.js
```
Comment (rgbkrk) [inline packages/webpack-configurator/copy_config.sh]: This seems reasonable as a step to perform.

File: packages/webpack-configurator/package.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "@nteract/webpack-configurator",
   "version": "2.0.0-alpha.0",
-  "main": "index.js",
+  "main": "lib/index.js",
   "description": "a common webpack configurator for nteract applications",
+  "files": ["../../babel.flow.config", "../../babel.flow.config"],
```
Comment (rgbkrk) [inline packages/webpack-configurator/package.json]: I don't think it'll be possible to grab and drop a file to a relative directory below. With the copy config above, have it run in a `prepare` script.

Comment (rgbkrk) [reply packages/webpack-configurator/package.json]: Oh I just realized this is in the webpack configurator...

File: packages/webpack-configurator/copy_config.sh (hunk 1 above)
Comment (jdetle) [reply packages/webpack-configurator/copy_config.sh]: I'm thinking that I should delete this with the `files` portion in package.json, but I'm pretty out of my element here.

File: packages/webpack-configurator/package.json (hunk 2 above)
Comment (jdetle) [reply packages/webpack-configurator/package.json]: I need to look into this more.

**Comment by jdetle** [comment]
@rgbkrk, @lgeiger I think this is ready to go, but I'm still not confident on how to test the distributed commuter app locally.

**Comment by rgbkrk** [comment]
I'll go ahead and bring this in, then we can figure out a good integration test.

**Pull request merged**